domain-cc/.pre-commit-config.yaml

```diff
@@ -0,0 +1,58 @@
+# Run pre-commit against all files by executing: pre-commit run --all-files
+# Auto-update the config to the latest repos' versions by executing: pre-commit autoupdate
+# Rather than disabling particular hooks (https://pre-commit.com/#temporarily-disabling-hooks), add an `exclude`
+
+# https://pre-commit.com/#regular-expressions
+exclude: >
+  (?x)^(
+      app/src/docker/docker-compose.yml
+  )
```

Comment:
Irrelevant. Leaving as a comment for reference:
```suggestion
# exclude: >
#  (?x)^(
#      app/src/docker/docker-compose.yml
#  )
```
or you can delete it.

Reply:
deleted